spec/integration/api/sys/quota_spec.rb

```diff
@@ -11,6 +11,7 @@
 
   it "creates a #{type} quota" do
     subject.sys.create_quota(type, "test_1", create_args)
+    p subject.sys.get_quota(type, "test_1")
```

Comment:
Not familiar with Ruby, wondering what the `p` is for?

Reply:
It's to print things... which I forgot to remove... thanks.

Reply:
Removed